### PR TITLE
VIH-6906 pexip error and offline error

### PR DESCRIPTION
**Pull request by levibutler**
### JIRA link (if applicable) ###
VIH-6906

### Change description ###
Error handling for pexip, due to the error being the same for user offline and pexip down ive had to implement a connection status check too.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by RocheL — DISMISSED** [review]

File: VideoWeb/VideoWeb/ClientApp/src/app/services/connection-status.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        // Assert
+        expect(XMLHttpRequest.prototype.open).toHaveBeenCalledTimes(2);
+        expect(XMLHttpRequest.prototype.send).toHaveBeenCalledTimes(2);
+    });
+
+    it('should stop timer if not started', () => {
+        expect(() => service.stopTimer()).not.toThrow();
+    });
+
+    it('should setup interval and fire intial check and again every 5 seconds', () => {
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/services/connection-status.service.spec.ts:54]: Not a big fan of a test that intentionally waits for 5+ seconds. The current test suit runs in half that time.

Comment (shaed-parkar) [reply VideoWeb/VideoWeb/ClientApp/src/app/services/connection-status.service.spec.ts:54]: Would you consider the ability to change the interval and then have then use the much smaller time period?

Comment (levibutler) [reply VideoWeb/VideoWeb/ClientApp/src/app/services/connection-status.service.spec.ts]: not waiting :) but yeah i wouldnt be either :P

**Review by shaed-parkar — APPROVED** [review]
LGTM

**Pull request merged**